CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change history for ui-notes
 
-## [10.0.0] IN PROGRESS
+## [10.0.0] (https://github.com/folio-org/ui-notes/tree/v10.0.0) (2024-10-30)
```

Comment:
Could you please add the diff like previous releases - [Full Changelog](https://github.com/folio-org/ui-notes/compare/v9.0.0...v9.1.0)